web/src/testgrid-grid-headers-block.ts

```diff
@@ -2,29 +2,31 @@ import { LitElement, html, css } from 'lit';
 import { customElement, property } from 'lit/decorators.js';
 import { map } from 'lit/directives/map.js';
 import { ListHeadersResponse } from './gen/pb/api/v1/data.js';
+// eslint-disable-next-line import/no-duplicates
 import { CombinedHeader } from './testgrid-grid-header-row.js';
+// eslint-disable-next-line import/no-duplicates
 import './testgrid-grid-header-row.js';
 
 function runLengthEncode(values: string[]): CombinedHeader[] {
-  if (values.length == 0) {
+  if (values.length === 0) {
     return [];
   }
 
   const headers: CombinedHeader[] = [];
   let prev = values[0];
   let count = 1;
-  for (let i = 1; i < values.length; i++) {
+  for (let i = 1; i < values.length; i += 1) {
     const curr = values[i];
-    if (prev == curr) {
-      count++;
-      continue;
```

Comment:
nit: I think I'd rather disable the 'no-continue' lint than use it. It seems like it's there because 'continue' obscures code flow, but I'm not sure I agree, and we use it frequently in Golang, so I feel like this'd be more familiar to most folks in the project anyways.

(I don't feel super strongly though so either way).

Reply:
Sounds good! (I also prefer the continue but not sure on JS/TS standards). I'll make the change this afternoon!